5539.txt

```diff
@@ -983,7 +983,7 @@ inspect them in the stadium, and they were now marching thither.
 
 He had ordered helmets, shields, and lances to be made after well-known
 Macedonian patterns and to be distributed to the new Hellenic legion.
-Later on they might be intrusted with the guarding of the city, should
+Later on they might be entrusted with the guarding of the city, should
```

Comment:
"intrusted" is an archaic spelling, but is appropriate to the original language, i.e. not a mistake.